examples/jsm/loaders/IfcLoader/IfcLoader.js

```diff
@@ -0,0 +1,125 @@
+//Example: https://github.com/tomvandig/web-ifc-three/tree/main/examples/jsm
+
+import { IfcAPI } from './web-ifc-api';
+import {
+  FileLoader,
+  Loader,
+  Object3D,
+  Mesh,
+  Color,
+  MeshPhongMaterial,
+  DoubleSide,
+  Matrix4,
+  BufferGeometry,
+  InterleavedBuffer,
+  InterleavedBufferAttribute,
+  BufferAttribute
+} from 'three/build/three.module';
```

Comment:
Unforunatelly, at the moment this needs to be `../../../../build/three.module.js`;